index.d.ts

```diff
@@ -6,6 +6,10 @@ type RequestInterface = typeof request;
 type RequestOptions = {
   request?: RequestInterface;
   token?: string;
+  cache?: {
+    id: string;
+    keys: VerificationPublicKey[];
+  };
```

Comment:
as we have that signature in a multiple places, I think we could define a new type such as `VerificationKeysCache` to remove the duplication